src/cisco_gnmi/client.py

```diff
@@ -98,6 +98,7 @@ def __init__(self, grpc_channel, timeout=_C_MAX_LONG, default_call_metadata=None
         """
         self.service = proto.gnmi_pb2_grpc.gNMIStub(grpc_channel)
         self.default_call_metadata = default_call_metadata
+        self._channel = grpc_channel
```

Comment:
Making it ``_channel`` marks it as a private entity which means the author intends it to only be used internally by the class.  You should either remove the ``_`` or create a function using the channel that you want users to use like maybe ``close()`` or ``disconnect()``.

Reply:
I don't necessarily encourage manipulating the `Channel` directly from the `Client`. It is a semi-internal deal - I'm not trying to mangle it with double underscore private `__channel`, but simultaneously don't want to think of it as a first class citizen owned by the `Client` class as the `Channel` could in theory be used by other gRPC services (such as gNOI).

Reply:
Do you mean multiple stubs, same channel?  Might want to design some reference counter associated to a channel.  I call xe.close() and when Client.__channel_count drops to zero, then call the actual channel.close().

Reply:
If we end up with a `cisco-gnxi-python` kind of library that's when we'd look at that kind of semantic, as is today the gNMI stub/`Client` class only use one channel so no need for counters. So, in the interest of not pretending like our `Client` implementation for gNMI actually owns the gRPC channel which will likely be shared, would like to not advertise the Channel reference too heavily outside of users that actually know what they're doing. There isn't a massive reason to be manually closing the channels AFAICT as they should just IDLE out.